src/predis/util.clj

```diff
@@ -1,5 +1,7 @@
 (ns predis.util)
 
+(defn id [x] x)
```

Comment:
This is already `clojure.core/identity`, but as stated before I think there's a better solution that doesn't need either